python/looker_sdk/__init__.py

```diff
@@ -47,21 +46,16 @@ def init31(
 ) -> methods31.Looker31SDK:
     """Default dependency configuration
     """
-    settings = api_settings.ApiSettings.configure(config_file, section)
-    if settings.api_version is not None:
-        warnings.warn(message=DeprecationWarning(API_SETTINGS_API_VERSION_DEPRECATED))
-    settings.api_version = "3.1"
-    settings.headers = {"Content-Type": "application/json"}
+    settings = api_settings.ApiSettings(config_file, section)
     if not settings.is_configured():
-        raise InitError(
-            f"Missing required configuration values like base_url and api_version."
-        )
+        raise InitError(f"Missing required configuration values.")
```

Comment:
This can be a normal string
```suggestion
        raise InitError(Missing required configuration values.")
```